### ISSUE TITLE
[BUG]: RSS Guard 4.2.2 can not be launched on macOS (no valid application)

### ISSUE THREAD

**Issue by nautavis** [martinrotter/rssguard#721]
### Brief description of the issue

RSS Guard 4.2.2 can not be launched, it is not recognized as a valid macOS application.
![Bildschirmfoto 2022-05-04 um 14 49 30](https://user-images.githubusercontent.com/49197786/166685162-4b0c53b5-7c97-462e-acbc-9652fd407403.png)

When trying to launch RSS Guard, a error message appears (translated, this is not the original English error message):
"RSS Guard.app can not be opened, it is possibly corrupted or invalid"

### How to reproduce the bug?

1. Download rssguard-4.2.2-4f2f991b-mac64.dmg or rssguard-4.2.2-4f2f991b-nowebengine-mac64.dmg
2. Open disk image (RSS Guard icon is missing)
3. Copy RSS Guard to Applications folder
4. Try to start RSS Guard

### What was the expected result?

RSS Guard launches

### What actually happened?

A macOS error message appears, saying "RSS Guard.app can not be opened, it is possibly corrupted or invalid"

### Debug log

No debug log available

### Operating system and version

* OS: macOS 12.3.1
* RSS Guard version: 4.2.2

**Comment by martinrotter** [comment]
app is not signed, could that be an issue?

**Comment by nautavis** [comment]
Probably not, because the icon should still be shown.
I just compared the contents of the 4.2.1 and 4.2.2 DMG file (disk image).
Compared to 4.2.1 the following files are missing in 4.2.2 (also shown in attached image):
- MacOS/rssguard
- Resources/rssguard.icns
- Info.plist
- Info.plist.in
- PkgInfo

Maybe something went wrong when this disk image was assembled?

![Bildschirmfoto 2022-05-10 um 09 10 05](https://user-images.githubusercontent.com/49197786/167571244-0d04eacf-e638-4346-8039-56b41afa7e78.png)

**Comment by martinrotter** [comment]
I see. I am not sure. Will check. I do not have macos machine so its kind of harder to debug for me.

**Comment by martinrotter** [comment]
Perhaps something was messed up. 4.2.1 works fine for you?

**Comment by nautavis** [comment]
Yes, 4.2.1 is working fine.
The latest dev. build is working. It is launchable, starts normally and I successfully fetched feeds.

**Comment by martinrotter** [comment]
Yes, in that case consider this fixed and feel free to use latest dev. build - the one should work fine. Something probably failed hard when it was building.

I likely fixed:

https://github.com/martinrotter/rssguard/commit/b9b1cb07d1eb74351d45e2fe5491b0b46db4c97a
https://github.com/martinrotter/rssguard/commit/5bad3b2db7310f1882984b5dd32281b036fc3247

Next stable version should be okay too.

**Comment by nautavis** [comment]
Thank you very much!
